cmd/cluster-controller/main.go

```diff
@@ -101,38 +103,72 @@ func main() {
 		fmt.Fprintln(os.Stderr, err)
 		os.Exit(1)
 	}
-	kcpSharedInformerFactory := kcpexternalversions.NewSharedInformerFactoryWithOptions(kcpclient.NewForConfigOrDie(r), resyncPeriod)
-	crdSharedInformerFactory := crdexternalversions.NewSharedInformerFactoryWithOptions(apiextensionsclient.NewForConfigOrDie(r), resyncPeriod)
 
-	apiImporterOptions := options.ApiImporterOptions.Complete(kubeconfig, kcpSharedInformerFactory, crdSharedInformerFactory)
-	apiImporter, err := apiImporterOptions.New()
+	crdClusterClient, err := apiextensionsclient.NewClusterForConfig(config)
```

Comment:
(No action required) Is there a reason we have `NewForConfigOrDie` but not `NewClusterForConfigOrDie`?